examples/platforms/efr32/Makefile.am

```diff
@@ -38,7 +38,11 @@ override CXXFLAGS                            := $(filter-out -Wconversion,$(CXXF
 override CFLAGS                              := $(filter-out -pedantic-errors,$(CFLAGS))
 override CXXFLAGS                            := $(filter-out -pedantic-errors,$(CXXFLAGS))
 
-EFR32MG_SDK_SRCDIR                            = $(top_srcdir)/third_party/silabs/gecko_sdk_suite/v2.0
+# Do not enable -Wimplicit-fallthrough for rail
+override CFLAGS                              := $(CFLAGS) -Wimplicit-fallthrough=0
+override CXXFLAGS                            := $(CXXFLAGS) -Wimplicit-fallthrough=0
+
```

Comment:
Had to disable implicit-fallthrough detector because kept getting

```
/home/tomas/Work/repos/openthread2/examples/../examples/platforms/efr32/../../../third_party/silabs/gecko_sdk_suite/v2.1/platform/emlib/src/em_cmu.c: In function 'CMU_ClockSelectSet':
/home/tomas/Work/repos/openthread2/examples/../examples/platforms/efr32/../../../third_party/silabs/gecko_sdk_suite/v2.1/platform/emlib/src/em_cmu.c:3491:14: error: this statement may fall through [-Werror=implicit-fallthrough=]
       selReg = (selReg == NULL) ? &CMU->LFACLKSEL : selReg;
       ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/tomas/Work/repos/openthread2/examples/../examples/platforms/efr32/../../../third_party/silabs/gecko_sdk_suite/v2.1/platform/emlib/src/em_cmu.c:3508:5: note: here
     case CMU_LFECLKSEL_REG:
     ^~~~
/home/tomas/Work/repos/openthread2/examples/../examples/platforms/efr32/../../../third_party/silabs/gecko_sdk_suite/v2.1/platform/emlib/src/em_cmu.c:3509:14: error: this statement may fall through [-Werror=implicit-fallthrough=]
       selReg = (selReg == NULL) ? &CMU->LFECLKSEL : selReg;
       ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/tomas/Work/repos/openthread2/examples/../examples/platforms/efr32/../../../third_party/silabs/gecko_sdk_suite/v2.1/platform/emlib/src/em_cmu.c:3516:5: note: here
     case CMU_LFBCLKSEL_REG:
     ^~~~
```